app/models/case/sar/offender.rb

```diff
@@ -60,10 +60,10 @@ def searchable_fields_and_ranks
                  third_party_company_name: :string,
                  late_team_id: :integer,
                  third_party_name: :string,
-                 number_dispatched_pages: :string,
+                 number_final_pages: :string,
                  number_exempt_pages: :string
```

Comment:
@willmcb  should number_final_pages and number_exempt_pages be integer?  then you don't have to check other possible values like string, so the logic for validating those values are more simpler

Reply:
Good idea will change in separate PR